### PR TITLE
[ticket/13578] - Add ucp_register.php event for use with third party ant...

### PR DESCRIPTION
**Pull request by RMcGirr83**
...i-spam services (eg, stopforumspam, akismet, etc).

PHPBB3-13578

### CODE REVIEW

**Comment by marc1706** [comment]
Commit message is too long. Please reduce it to a maximum of 80 characters per line.

**Comment by marc1706** [comment]
Also, the PR should be against develop-ascraeus and not develop.

File: phpBB/includes/ucp/ucp_register.php [hunk 1]
```diff
@@ -259,6 +259,16 @@ function main($id, $mode)
 					$error[] = $user->lang['NEW_PASSWORD_ERROR'];
 				}
 			}
+			/**
+			* Add UCP registration data before they are submitted
+			*
+			* @event core.ucp_register_data
+			* @var	array	data		Array with current ucp registration data
+			* @var	array	error		Array with list of errors
+			* @since 3.1.3
```
Comment (marc1706) [inline phpBB/includes/ucp/ucp_register.php]: Should be 3.1.4-RC1

**Comment by RMcGirr83** [comment]
Hi Marc,

Pretty sure I have to close this PR to PR to correct branch(?).  So I will close and re-do to correct branch and with correct number of characters as requested and if needed.

Also seems the PR is a bit messed up as the commit isn't displaying correctly for me.

**Comment by marc1706** [comment]
Correct, you'll have to create another PR.

**Pull request closed without merging**